openr/py/openr/cli/utils/utils.py

```diff
@@ -23,9 +22,7 @@
 
 import bunch
 import click
-from libfb.py.decorators import memoize_forever
 from openr.AllocPrefix import ttypes as alloc_types
-from openr.clients.openr_client import get_openr_ctrl_client
```

Comment:
I thought you'd fix thrift py3 clients support.

Reply:
I did but the resultant shared objects are too big for TG